routemaster/conftest.py

```diff
@@ -280,9 +314,9 @@ def _mock(data={'should_do_alternate_action': False}):
 
 
 @pytest.fixture()
-def assert_history():
+def assert_history(app_config):
```

Comment:
This means that we won't be able to use this fixture if a custom config has been specified. Is that intentional?

Reply:
That's true, but we don't do that much and don't do it at all at the call sites of this util. We can always move it back if needed.